src/index.js

```diff
@@ -4,6 +4,7 @@ import cors from 'cors';
 import debug from 'debug';
 import messages from './utils/messages';
 import response from './utils/response';
+import _ from './config/env';
```

Comment:
Please can you explain why you have this config import as `_`
I'm sorry but I don't know.

Reply:
it replaces
```javascript
import dotenv from 'dotenv';
dotenv.config();
```
since nothing is being exported, the `_` prevents _unused import_ warning.

Reply:
Okay cool, thanks for the explanation.

Reply:
You're welcome.